getting-started/welcome-to-openadapt/quick-start.md

```diff
@@ -47,48 +58,83 @@ pip install -r requirements.txt
 pip install -e .
 alembic upgrade head
 ```
-{% endtab %}
 
-{% tab title="pip" %}
-```
-pip install openadapt
-```
 {% endtab %}
 
-{% tab title="Installer" %}
-Download / Clone OpenAdapt from [here](https://github.com/MLDSAI/OpenAdapt), then run one of the following scripts:\
-Windows: [install\_openadapt.ps1](https://github.com/MLDSAI/OpenAdapt/blob/main/install/install\_openadapt.ps1)\
-MacOS:  [install\_openadapt.sh](https://github.com/MLDSAI/OpenAdapt/blob/main/install/install\_openadapt.sh)
+{% tab title="Scripted Installation" %}
+[Download](https://openadapt.ai/#start)
+
+# Windows:
+
+- Press Windows Key, type "powershell", and press Enter
+- Copy and paste the following command into the terminal, and press Enter (If Prompted for `User Account Control`, click 'Yes'):
+- `Start-Process powershell -Verb RunAs -ArgumentList '-NoExit', '-ExecutionPolicy', 'Bypass', '-Command', "iwr -UseBasicParsing -Uri 'https://raw.githubusercontent.com/OpenAdaptAI/OpenAdapt/main/install/install_openadapt.ps1' | Invoke-Expression"`
+
+# MacOS:
+
+- Download and install Git and Python 3.10
+- Press Command+Space, type "terminal", and press Enter
+- Copy and paste the following command into the terminal, and press Enter:
+  `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/OpenAdaptAI/OpenAdapt/HEAD/install/install_openadapt.sh)"`
+
 {% endtab %}
 {% endtabs %}
 
+# Permissions(IMPORTANT)
+
+See how to set up system permissions on macOS [here](https://github.com/OpenAdaptAI/OpenAdapt/blob/main/permissions_in_macOS.md).
+
 {% hint style="info" %}
 **Good to know:** After installing OpenAdapt, run\
 `pytest` to verify that the installation was successful.
 {% endhint %}
 
-## Make your first recording
+## Usage
```

Comment:
Please move this to a new section titled something like "Developer Usage".

The goal of this exercise is to have documentation for *users*, i.e. people with no command line experience.

I would expect to see the following rough breakdown:

A: Download and install
1. Download OpenAdapt
2. Install OpenAdapt (for now this is just extracting the zip)
3. Run OpenAdapt
5. Complete the onboarding (this is almost merged, https://github.com/OpenAdaptAI/OpenAdapt/pull/746)

B: Create a recording
1. Right-click the tray icon -> Click "Record" -> Type a task description -> Click "OK"
2. Open some apps, type some letters, move/click/scroll the mouse
3. Stop the recording by pressing "Ctrl" three times
4. Wait for it to stop

C: View the recording
1. Right-click the tray icon -> click "Launch Dashboard" (if it isn't already running)
2. Click "Recordings"
3. Click on the recording you just created
4. Scroll down through the events, removing any accidental/unnecessary actions by clicking "Delete"

D: Replay the recording
1. Right-click the tray icon -> click "Replay" -> Select the recording
2. Select a replay strategy (leaving the default VisualReplayStrategy is recommended)
3. Type some text instructions
4. Click "OK"

E: Upload your recording and replay
1. Coming soon! (https://github.com/OpenAdaptAI/OpenAdapt/issues/767)

Reply:
Hi @abrichr 
For me downloading the app and installing gets stuck in terminal here(and i tried this multiple times). And there is no prompt that comes up

<img width="1458" alt="image" src="https://github.com/OpenAdaptAI/OpenAdapt/assets/48802836/ba185084-3c18-4962-9d6b-fe831d1edd02">

For me command line worked best with `start` command

Is there a better way to start the app? Because i gave all the necessary Mac permissions needed for it.

Reply:
Hi @shashank40 , sorry to hear that, thank you for letting us know. Can you please provide the full console output? @KIRA009 any suggestions would be appreciated 🙏

Reply:
Can you please provide the exact command (or ideally sequence of commands) you ran in order to arrive at this error?

Reply:
No @KIRA009 i see none. I was looking at the reasons why it is not coming for me, but i couldn't find any robust reason.

Reply:
Yesss @KIRA009 i downloaded it today. I will re-download it

Reply:
@KIRA009 not working for me

Reply:
Working fine for me now. I was able to debug it.

So basically we might need to update the permission section for latest macOS.

It needs to get screen-recording permission for OpenAdapter app as well.

Works fine for me now. Just need to get the latest MAC build, that might get updated in sometime if this PR
 https://github.com/OpenAdaptAI/OpenAdapt/pull/776 is approved and ready to get merged

Reply:
Thank you @shashank40 ! Tracked in https://github.com/OpenAdaptAI/OpenAdapt/issues/778